tests/main_test.py

```diff
@@ -4,11 +4,22 @@
 from datetime import datetime
 import src.main as main
 
+
+def get_wel(lan_code: str) -> str:
+
+    with open("src/welcome.json", "r") as f:
+        wlc_mess_list = main.load(f)[lan_code]
+    
+    #first element of the list
+    return wlc_mess_list[0].replace("USER","@user")
```

Comment:
This handles only the case in which the user has an username, in case the `possible_users` array will be modified, this test could fail